k8s/admissionregistration/validatingwebhooks_v1beta1.go

```diff
@@ -0,0 +1,52 @@
+package admissionregistration
+
+import (
+	"context"
+
+	hook "k8s.io/api/admissionregistration/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ValidatingWebhookConfigurationV1beta1Ops is interface to perform CRUD ops on mutatting webhook controller
+type ValidatingWebhookConfigurationV1beta1Ops interface {
+	// GetValidatingWebhookConfigurationV1beta1 returns a given ValidatingWebhookConfiguration
+	GetValidatingWebhookConfigurationV1beta1(name string) (*hook.ValidatingWebhookConfiguration, error)
+	// CreateValidatingWebhookConfigurationV1beta1 creates given ValidatingWebhookConfiguration
+	CreateValidatingWebhookConfigurationV1beta1(req *hook.ValidatingWebhookConfiguration) (*hook.ValidatingWebhookConfiguration, error)
+	// UpdateValidatingWebhookConfigurationV1beta1 updates given ValidatingWebhookConfiguration
+	UpdateValidatingWebhookConfigurationV1beta1(*hook.ValidatingWebhookConfiguration) (*hook.ValidatingWebhookConfiguration, error)
```

Comment:
Can you either add req to both functions or remove from both function please, let's be consistent

Reply:
✅